src/Microsoft.ApiDesignGuidelines.Analyzers/Microsoft.ApiDesignGuidelines.Analyzers.md

```diff
@@ -190,6 +298,26 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182243.aspx](http://msdn.microsoft.com/library/ms182243.aspx)
 
+### CA1716: Identifiers should not match keywords ###
+
+A namespace name or a type name matches a reserved keyword in a programming language. Identifiers for namespaces and types should not match keywords that are defined by languages that target the common language runtime.
+
```

Comment:
This rule also applies to the names of publicly visible virtual class members and interface members, and to the names of the parameters of those members. (We've had a lot of discussion about why the rule is limited to virtual and interface methods. It's just as much of a problem for non-virtual methods. But that's moot for purposes of this PR.)